ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/SignTransactionModal.qml

```diff
@@ -0,0 +1,190 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import QtQuick.Dialogs 1.3
+import "../../../../../imports"
+import "../../../../../shared"
+import "../../../../../shared/status"
+
+ModalPopup {
+    property var selectedAccount
+    property var selectedRecipient
+    property var selectedAsset
+    property var selectedAmount
+    property var selectedFiatAmount
+    property var selectedGasLimit
+    property var selectedGasPrice
+
+    id: root
+
+    //% "Send"
+    title: qsTrId("command-button-send")
+    height: 504
+
+    property MessageDialog sendingError: MessageDialog {
+        id: sendingError
+        title: qsTr("Error sending the transaction")
+        icon: StandardIcon.Critical
+        standardButtons: StandardButton.Ok
+    }
+    property MessageDialog sendingSuccess: MessageDialog {
+        id: sendingSuccess
+        //% "Success sending the transaction"
+        title: qsTrId("success-sending-the-transaction")
+        icon: StandardIcon.NoIcon
+        standardButtons: StandardButton.Ok
+        onAccepted: {
+            root.close()
+        }
+    }
+
+    onClosed: {
+        stack.reset()
+    }
+
+    function sendTransaction() {
+        let responseStr = walletModel.sendTransaction(root.selectedAccount.address,
+                                                 root.selectedRecipient.address,
+                                                 root.selectedAsset.address,
+                                                 root.selectedAmount,
+                                                 gasSelector.selectedGasLimit,
+                                                 gasSelector.selectedGasPrice,
+                                                 transactionSigner.enteredPassword)
+        let response = JSON.parse(responseStr)
+
+        if (response.error) {
+            if (response.error.includes("could not decrypt key with given password")){
+                transactionSigner.validationError = qsTr("Wrong password")
+                return
+            }
+            sendingError.text = response.error
+            return sendingError.open()
+        }
```

Comment:
Also updated in the PR that was merged after this PR was created, `sendTransaction` returns an `RpcResponse` object, so this can be updated to:
```suggestion
        if (response.error) {
            if (response.error.message.includes("could not decrypt key with given password")){
                transactionSigner.validationError = qsTr("Wrong password")
                return
            }
            sendingError.text = response.error.message
            return sendingError.open()
        }
```